### PR TITLE
qdevices: Adapt virtiofsd options

### PR DESCRIPTION
**Pull request by hellohellenmao**
Some options are deprecated since the dedicated virtiofsd package since rhel9, while before virtiofsd belongs to qemu package.

### CODE REVIEW

**Comment by hellohellenmao** [comment]
@luckyh @YongxueHong Could you please help to take a review here? Thanks.

**Comment by hellohellenmao** [comment]
The most basic case passed on both old version and new version.
Old version:
(2/2) Host_RHEL.m9.u2.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.x86_64.io-github-autotest-qemu.virtio_fs_share_data.with_cache.auto.q35: PASS (67.54 s)
New version:
 (1/1) Host_RHEL.m9.u5.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.5.0.x86_64.io-github-autotest-qemu.virtio_fs_share_data.with_cache.auto.q35: PASS (66.26 s)

**Comment by hellohellenmao** [comment]
@luckyh Could you please help to take a review here? thanks.

File: virttest/qemu_devices/qdevices.py [hunk 1]
```diff
@@ -2259,11 +2260,27 @@ def _handle_log(self, line):
...
+        virtiofs_binary = self.get_param("binary")
+        socket_path = self.get_param("sock_path")
+        source_dir = self.get_param("source")
+
+        fsd_cmd = f"{virtiofs_binary} --socket-path={socket_path}"
+
+        # Capture the capabilities of virtiofsd
+        result = subprocess.check_output(
+            [virtiofs_binary, "--print-capabilities"], universal_newlines=True
         )
```
[the remark this answers is not in the document]
Comment (hellohellenmao) [reply virttest/qemu_devices/qdevices.py]: Updated. @luckyh Could you please take another review here? Thanks.

**Comment by hellohellenmao** [comment]
@YongxueHong Could you please help to review here? Thanks.

**Comment by hellohellenmao** [comment]
The basic caset passed with the latest code:
 (1/1) Host_RHEL.m9.u5.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.5.0.x86_64.io-github-autotest-qemu.virtio_fs_share_data.with_cache.auto.q35: PASS (66.17 s)

**Review by luckyh — APPROVED** [review]
LGTM

**Review by YongxueHong — APPROVED** [review]
LGTM.

**Pull request merged**